external/radiation/python/radiation_driver.py

```diff
@@ -747,31 +747,9 @@ def _GFS_radiation_driver(
                     if ccnd[i, k, 0] < self.EPSQ:
                         ccnd[i, k, 0] = 0.0
 
-        if Model["uni_cld"]:
```

Comment:
Which of these options is not implemented? 

It appears Andrew only partially ported the `Model["effr_in"]` logic, since effrl, etc are not used anywhere. This PR would be clearer if it deleted all the dead code e.g. references to the `effrl`, `effrr`, etc and raised an error near the top of the function if `Model["effr_in"]` is true.

Reply:
Model["effr_in"] is set to False but I added exception error for when it is set to True.